editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -1096,24 +1096,27 @@ Skeleton3DEditor::~Skeleton3DEditor() {
 	Node3DEditor *ne = Node3DEditor::get_singleton();
 
 	if (animation_hb) {
-		ne->remove_control_from_menu_panel(animation_hb);
+		topmenu_bar->remove_child(animation_hb);
 		memdelete(animation_hb);
 	}
 
 	if (separator) {
-		ne->remove_control_from_menu_panel(separator);
+		topmenu_bar->remove_child(separator);
 		memdelete(separator);
 	}
 
 	if (skeleton_options) {
-		ne->remove_control_from_menu_panel(skeleton_options);
+		topmenu_bar->remove_child(skeleton_options);
 		memdelete(skeleton_options);
 	}
 
 	if (edit_mode_button) {
-		ne->remove_control_from_menu_panel(edit_mode_button);
+		topmenu_bar->remove_child(edit_mode_button);
 		memdelete(edit_mode_button);
 	}
```

Comment:
Children will automatically be deleted when the parent is deleted.

These separate `remove_child()` and `memdelete()` calls won't be necessary anymore :)